src/core_plugins/kibana/public/discover/index.html

```diff
@@ -5,14 +5,14 @@
     <div data-transclude-slots>
       <!-- Breadcrumbs. -->
       <div data-transclude-slot="topLeftCorner" class="kuiLocalBreadcrumbs">
-        <div class="kuiLocalBreadcrumb">
+        <h1 tabindex="0" id="kui_local_breadcrumb" class="kuiLocalBreadcrumb">
```

Comment:
The id is required when you have a tabindex=0 for some reason. It's just a weird property of the platform. You can test it yourself by removing the ID. I think it's because the expectation is that you could nav there with #kui_local_breadcrumb in the url